js/app.js

```diff
@@ -0,0 +1,75 @@
+function lengths (strs) {
+	var sizes = [];
+	// console.log(strs);
+	for(string in strs) {
+		sizes.push(strs[string].length);
+	}
+	return sizes;
+}
+
+function transmogrifier(a, b, c) {
+	if(arguments.length != 3) {
+		console.log ("THREE NUMBERS DUDE.  NO MORE NO LESS");
+		return;
+	}
+	return (a + b)**c;
+}
+
+function toonify(accent, sentence) {
+	console.log("DUDE REGULAR EXPRESSIONS ARE A FRICKIN RABBIT HOLE");
+	if(accent=="daffy") {
+		return sentence.replace(/s/gi, 'th');
+	}
+	if(accent=="elmer") {
+		return sentence.replace(/r/gi, 'w');
+	}
+	if(accent=="elongated") {
+		return sentence.replace(/[A-Z]/gi, '$&$&');
+	}
+	if(accent=="elongated vowels") {
+		return sentence.replace(/[A,o,e,i,u]/gi, '$&$&');
+	}
+
+	else return "no accent specified";
+}
+
+function wordReverse(str) {
+	// var wordArray = str.split(' ');	
+	// console.log(wordArray);
+	var revArray = (str.split(' ')).reverse();
+	var returnString = "";
+	for(var i = 0; i<revArray.length; i++) {
+		returnString += revArray[i];
+		if(i==revArray.length-1){
```

Comment:
ah yea of coursed-- like add the space if it's not = 1 and get rid of the else statement